.github/workflows/test-universal-html2parquet.yml

```diff
@@ -0,0 +1,128 @@
+#
+# DO NOT EDIT THIS FILE: it is generated from test-transform.template,  Edit there and run make to change these files
+#
+name: Test - transforms/universal/html2parquet
```

Comment:
Did't we move this to language ? or maybe my brain is tired... either case, better block the merge for now. Sorry

Reply:
Nice catch.  This happens because git does not seem to remove directories/files when switching to branches w/o them.  Something to watch out for.